core-bundle/src/EventListener/DbafsMetadataSubscriber.php

```diff
@@ -57,6 +59,12 @@ public function enhanceMetadata(RetrieveDbafsMetadataEvent $event): void
             $event->set('importantPart', $importantPart);
         }
 
+        // Add subtitles information
+        if (!empty($language = $row['subtitlesLanguage'] ?? null)) {
+            $subtitles = new Subtitles($language, $row['subtitlesType'] ? SubtitlesType::from($row['subtitlesType']) : null);
```

Comment:
You can use [BackedEnum::tryFrom](https://www.php.net/manual/en/backedenum.tryfrom.php) here.